cocos/loop/Collections.h

```diff
@@ -0,0 +1,139 @@
+/****************************************************************************
+Copyright (c) 2008-2010 Ricardo Quesada
+Copyright (c) 2009      Valentin Milea
+Copyright (c) 2010-2012 cocos2d-x.org
+Copyright (c) 2011      Zynga Inc.
+Copyright (c) 2013-2016 Chukong Technologies Inc.
+Copyright (c) 2017-2018 Xiamen Yaji Software Co., Ltd.
```

Comment:
The Copyright is `Copyright (c) 2018 Xiamen Yaji Software Co., Ltd.`, it is nothing about other person or companies.